### ISSUE TITLE
Add config for gear and tools damage and armor values [Enhancement]

### ISSUE THREAD

**Issue by panbanann** [Luligabi1/MagicFungi#56]
Hi,
Is it possible you could add the config for armours and tools/weapons that will allow balancing the values by modpack developers?
The mod is really nice addition to our pack but it is really overpowering the rest of the items.
Maybe also add support for disabling certain items in config as well.
I think it should be a good addition to the mod.

**Comment by Luligabi1** [comment]
All the relics have config values to modify the strength and time of their special effects. If neccessary, I can add a value to set their durability too. Would that help?


Which items do you find should have a disable option?

**Comment by panbanann** [comment]
Thanks for a fast reply.
We are using version 0.7.0 for mc 1.18.1 and the only configs available are for spell effects and cooldowns so maybe that is why it is a problem. Any chance to port back the config improvements to 1.18.1?
The problem we have is with a scythe, sword damage and armour value for armours would be nice to adjust for balance purposes.
For disabling it would be nice to be able to disable anything. Different modpacks could benefit from that. The main problem is with the impetus sword being overpower af and it costs nothing, so if modpack devs could adjust the value of damage to match their need it would be nice.

**Comment by Luligabi1** [comment]
> Thanks for a fast reply. We are using version 0.7.0 for mc 1.18.1 and the only configs available are for spell effects and cooldowns so maybe that is why it is a problem. Any chance to port back the config improvements to 1.18.1? The problem we have is with a scythe, sword damage and armour value for armours would be nice to adjust for balance purposes.

How long since the last time you have regenerated the config values? 0.7.0 already has most of config values I've mentioned, in specific, it should have:

```
utilisPickaxeEffectDuration
utilisPickaxeEffectStrength


morbusScytheEffectDuration
morbusScytheEffectStrength;


magicalFungiArmorDurabilityMultiplier
magicalFungiArmorToughness
magicalFungiKnockBackResistance
```

> For disabling it would be nice to be able to disable anything. Different modpacks could benefit from that. The main problem is with the impetus sword being overpower af and it costs nothing, so if modpack devs could adjust the value of damage to match their need it would be nice.

I usually let these balance issues up to modpack developers as it's much easier to modify/remove recipes with KubeJS than to me add an option to do so. I'll consider the damage config values, but they'll probably not be available on 1.18.1 as the  current target version is 1.18.2 🙁

**Comment by panbanann** [comment]
Ok, thanks for the info, our bad. I just refreshed the config and have all the new stuff. I wasn't aware that the server owner copied old configs from the previous version.
Sorry for the trouble.

**Comment by Luligabi1** [comment]
No problem! Glad I could help. I'll keep this open as a reminder to add the other config options.

**Comment by Luligabi1** [comment]
Implemented on `7c76c3b`, adding the following values:

```
relicDurability
relicMiningSpeed

impetusSwordAttackDamage 
morbusScytheAttackDamage
```

Hopefully this adresses the issues you had. Thanks for contributing! The update including these changes will come out in a few days.
